### PR TITLE
Update alexa.intent.markdown

### PR DESCRIPTION
**Pull request by cfreeze**
Updated with changes in the AWS and Alexa UIs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @cfreeze, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21308).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/alexa.intent.markdown [hunk 1]
```diff
@@ -80,7 +78,9 @@ Next you need to create a Lambda function.
...
 - Save your environmental variables by clicking the `Save` button.
-- Next, copy the ARN displayed in the top of the page, which is the identify of this Lambda function. Set the end point of the custom Alexa Skill you created earlier to this value.
+- Next, copy the ARN displayed in the top of the page, which is the identify of this Lambda function.
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]:
```suggestion
- Next, copy the ARN displayed at the top of the page, which is the identity of this Lambda function. 
```

File: source/_integrations/alexa.intent.markdown [hunk 2]
```diff
@@ -80,7 +78,9 @@ Next you need to create a Lambda function.
...
 - Save your environmental variables by clicking the `Save` button.
-- Next, copy the ARN displayed in the top of the page, which is the identify of this Lambda function. Set the end point of the custom Alexa Skill you created earlier to this value.
+- Next, copy the ARN displayed in the top of the page, which is the identify of this Lambda function. 
+  - Go back to your Alexa skill and go to Custom->Endpoint menu option on the left.
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]:
```suggestion
  - Go back to your Alexa skill and go to the Custom->Endpoint menu option on the left.
```

File: source/_integrations/alexa.intent.markdown [hunk 3]
```diff
@@ -80,7 +78,9 @@ Next you need to create a Lambda function.
...
+  - Paste the ARN value in the "Default Region".  Note: you will not be able to do this until you have completed the step above adding the Alexa Skills Kit trigger (done in previous step) to the AWS Lambda Function.
```
Comment (frenck) [inline source/_integrations/alexa.intent.markdown]:
```suggestion
  - Paste the ARN value in the "Default Region". Note: you will not be able to do this until you have completed the step above adding the Alexa Skills Kit trigger (done in the previous step) to the AWS Lambda Function.
```

**Review by frenck — APPROVED** [review]
Thanks, @cfreeze 👍

**Pull request merged**